login.html

```diff
@@ -0,0 +1,36 @@
+<html>
```

Comment:
```suggestion
<html lang="en">
```

(It's an accessibility thing)